sklearn/covariance/tests/test_graph_lasso.py

```diff
@@ -69,9 +127,9 @@ def test_graph_lasso_cv(random_state=1):
     try:
         sys.stdout = StringIO()
         # We need verbose very high so that Parallel prints on stdout
-        GraphLassoCV(verbose=100, alphas=3, tol=1e-1).fit(X)
+        GraphLassoCV(verbose=100, alphas=5, tol=tol).fit(X)
```

Comment:
The alphas change is mentioned in the commit message: for widely-spaced alphas, the warm starts used by the CV loop are very bad, which actually can lead to lack of convergence (see http://web.stanford.edu/~hastie/Papers/glassoinsights.pdf).

Tolerance: the tolerance parameter is funny because the same value is used for both the overall glasso algorithm and for the elastic net solver inside the main loop. So even if we are OK with a high tolerance for Theta, passing one in here can cause the algorithm to fail to converge because the (lasso) steps are very far from optimal.